engine.py

```diff
@@ -269,34 +312,26 @@ def show_panel(self, panel_id, title, bundle, widget_class, *args, **kwargs):
                     break
     
             if existing_pane is None and nuke.env.get("NukeVersionMajor") < 9:
-                # couldn't find anything to parent next to!
-                # nuke 9 will automatically handle this situation
+                # Couldn't find anything to parent next to!
+                # Nuke 9 will automatically handle this situation
                 # but older versions will not show the UI!
-                # tell the user that they need to have the property
-                # pane present in the UI
+                # Tell the user that they need to have the property
+                # pane present in the UI.
                 nuke.message("Cannot find any of the standard Nuke UI panels to anchor against. "
                              "Please add a Properties Bin to your Nuke UI layout and try again.")
                 return
     
-            # ok all good - we are running nuke 9 and/or 
+            # All good - we are running nuke 9 and/or 
             # have existing panes to parent against.
             panel_widget.addToPane(existing_pane)   
-        
         else:
-            # we are either calling this from a pane restore
+            # We are either calling this from a pane restore
             # callback or from the pane menu in Nuke. In both
             # these cases, the current pane is already established
             # by the system, so just add our widget.
-            # add it to the current pane
+            # Add it to the current pane
             panel_widget.addToPane()
-
-        
-        # return the created widget
-        return panel_widget     
-        
-    
-    ##########################################################################################
-    # managing favorite dirs            
+        return panel_widget                
     
     def __setup_favorite_dirs(self):
```

Comment:
so there seems to be a bunch of separate nuke and hiero methods and logic. Depending on the direction we are going with the final engine, wondering if it's worth splitting this up in two separate python files, one for nuke and one for hiero, for simplicity? But maybe that's overkill? Or maybe that's counter productive given the direction we are headed with a fully merged engine?